src/operator-docs/operator.model.ts

```diff
@@ -22,11 +23,11 @@ export interface OperatorParameters {
   name: string;
   type: string;
   attribute: string;
-  description: string;
+  description: any;
```

Comment:
maybe it makes sense to replace any with a complex object which holds a property for each supported language. This would make it really easy to see which languages are supported and maybe missing for future PRs. Furthermore this object could be used for a language choser or something like that.